Kickstarter-iOS/Views/Controllers/FeatureFlagToolsViewController.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class FeatureFlagToolsViewController: UITableViewController {
+  private let viewModel = FeatureFlagToolsViewModel()
+
+  // MARK: - Properties
+
+  private var features = [(Feature, Bool)]()
+  private let reuseId = "FeatureFlagTools.TableViewCell"
+
+  static func insantiate() -> FeatureFlagToolsViewController {
+    return FeatureFlagToolsViewController.init(nibName: nil, bundle: nil)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self.tableView
+      |> \.tableFooterView .~ UIView(frame: .zero)
+
+    self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: self.reuseId)
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.reloadWithData
+      .observeForUI()
+      .observeValues { [weak self] features in
+        self?.features = features
+
+        self?.tableView.reloadData()
+      }
+
+    self.viewModel.outputs.updateConfigWithFeatures
+      .observeForUI()
+      .observeValues { [weak self] features in
+        self?.updateConfig(with: features)
+      }
+  }
+
+  @objc func switchTogged(_ switchControl: UISwitch) {
+    self.viewModel.inputs.setFeatureAtIndexEnabled(index: switchControl.tag, isEnabled: switchControl.isOn)
+  }
+
+  // MARK: - Private Helpers
+
+  private func updateConfig(with features: Features) {
+    guard let config = AppEnvironment.current.config else { return }
+
+    let updatedConfig = config |> \.features .~ features
+
+    AppEnvironment.updateConfig(updatedConfig)
+
+    self.viewModel.inputs.didUpdateConfig()
+  }
+}
+
+// MARK: - UITableViewDataSource
+
+extension FeatureFlagToolsViewController {
+  override func numberOfSections(in _: UITableView) -> Int {
+    return 1
```

Comment:
This should be a default number of sections so this data source function can be omitted.